tests/components/test_producer_app.py

```diff
@@ -352,3 +361,72 @@ def test_get_output_topics(
             KafkaTopic(name="producer-app-output-topic"),
             KafkaTopic(name="extra-topic-1"),
         ]
+
+    @pytest.mark.asyncio()
+    async def test_should_deploy_clean_up_job_with_values_in_cluster_when_clean(
```

Comment:
should we add a test for reset as well?

Reply:
The producer app doesn't have a reset, but I added a test to check it 😄